src/highlight/HighlightManager.tsx

```diff
@@ -40,7 +47,7 @@ export default class HighlightManager implements BaseManager {
     }
 
     render(props: Props): void {
-        ReactDOM.render(<HighlightContainer location={this.location} {...props} />, this.reactEl);
+        ReactDOM.render(<HighlightContainer location={this.location} pageEl={this.pageEl} {...props} />, this.reactEl);
```

Comment:
What do you think about using the `referenceEl` passed in already? That should most likely be the textLayer which is the only layer that would trigger mousedown/up events for text selection

Reply:
Waiting for product/design decision